### PR TITLE
User Request continuted

### PR DESCRIPTION
**Pull request by ChineduOguejiofor**
#### What does this PR achieve?
- This is the second Part of the first PR

### CODE REVIEW

File: src/components/userRequestService/components/Description.jsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+const { NAIRA, AN_X } = STRINGS;
+const columns = [
+    {
+        dataIndex: 'garment',
+        key: 'garment',
+        render: (text, record) => (
+            <>
+                {record.garment.split(' ')[0] === 'Grand'
+                    ? (
+                        <span style={{ fontWeight: 'bold' }}>
```
Comment (edbella) [inline src/components/userRequestService/components/Description.jsx]: Why not use a Fragment?

File: src/components/userRequestService/components/Description.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                            {record.garment}
+                        </span>
+                    )
+                    : (
+                        <span>
```
Comment (edbella) [inline src/components/userRequestService/components/Description.jsx]: Same here as above.

File: src/components/userRequestService/components/Description.jsx (hunk 1 above)
Comment (justiceotuya) [reply src/components/userRequestService/components/Description.jsx]: That's the short form of Fragment @edbella

File: src/components/userRequestService/components/Description.jsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                {record.garment.split(' ')[0] === 'Grand'
+                    ? (
+                        <span style={{ fontWeight: 'bold' }}>
+                            {NAIRA}
+                            {' '}
+                            {record.garmentPrice * record.grandTotal}
+                        </span>
+                    )
+                    : (
+                        <span>
```
Comment (edbella) [inline src/components/userRequestService/components/Description.jsx]: Use a fragment since you're not styling

Comment (justiceotuya) [reply src/components/userRequestService/components/Description.jsx]: https://reactjs.org/docs/fragments.html#short-syntax

File: src/components/userRequestService/components/Description.jsx (hunk 2 above)
Comment (olajohn-ajiboye) [reply src/components/userRequestService/components/Description.jsx]: That's just the shorthand. I think it's a preference thing.

Comment (olajohn-ajiboye) [reply src/components/userRequestService/components/Description.jsx]: I would though encapsulate the render function into a reusable function. Think something that accepts the `record` and `text` as it is, but returns span based on a third argument like the `dataIndex`. You would have a cleaner render function and no repetition. . You will notice the first few lines are the same.

Comment (ChineduOguejiofor) [reply src/components/userRequestService/components/Description.jsx]: Okay thanks

**Pull request merged**